### ISSUE TITLE
Encoding efficiency regression

### ISSUE THREAD

**Issue by EwoutH** [OpenVisualCloud/SVT-AV1#401]
Somewhere between 6e93de7 (June 10) and ac75c3e a slight coding efficiency regression took place. I can't find any commits that reduced enocding performance (in favor of speed for example), was this expected?

[BD rate increases](https://beta.arewecompressedyet.com/?job=SVT-AV1-enc-mode-8%402019-06-10&job=SVT-AV1-enc-mode-8%402019-06-21):

|  | PSNR | PSNR HVS | SSIM | CIEDE 2000 | PSNR Cb | PSNR Cr | APSNR | APSNR Cb | APSNR Cr | MS SSIM |
|---------|:----:|:--------:|:----:|:----------:|:-------:|:-------:|:-----:|:--------:|:--------:|:-------:|
| Average | 0.43 |   0.44   | 0.46 |    2.24    |   6.05  |   4.97  |  0.43 |   6.12   |   4.99   |   0.45  |

**Comment by gnafuthegreat** [comment]
While that is worrying if unexpected, it does look like speed increased significantly for all Q levels aside from 43.  Nearly 10-20% speedup at various Q levels seems like a reasonable tradeoff for the fastest speed setting.  But I agree it would be nice to confirm when this happened and whether it's expected :-).

**Comment by gnafuthegreat** [comment]
Oh, wait.  I see it actually got both slower _and_ worse quality at speed 3:

https://beta.arewecompressedyet.com/?job=SVT-AV1-enc-mode-3%402019-06-10&job=SVT-AV1-enc-mode-3%402019-06-21

**Comment by EwoutH** [comment]
@hassount I would tag this as P1 and put someone on it. The longer this remains unsolved the more difficult it is to track back. 0.5% is not insignificant.

**Comment by hassount** [comment]
@EwoutH yes this is being worked on as part of #379

**Comment by EwoutH** [comment]
> yes this is being worked on as part of #379

Sorry but this goes against all open source development practices and even common sense. It's impossible to test if this regression has been fixed if it's part of a larger PR. That makes it very likely that while the PR would improve the performance, it could have been even better if this regression was fixed first. You just don't know if the regression is still there or not.

I advise strongly to follow the following steps:
- Detect that there is a problem (done)
- Find the moment when the problem first occurred (the exact commit)
- Find out the source of the problem (what causes the regression)
- Revert or fix it
- Test that the problem is solved

**Comment by hassount** [comment]
@EwoutH 
Thank you for the feedback. My comment was not clear enough so let me rephrase. 
As part of #379  we first make sure that the regression issues such as this one is addressed before we start working on creating new presets. Right now, we are trying to stabilize the encoder to even be able reproduce this data, because we can not reproduce the above on our side yet. Also the speed for AWCY is measured on a single process execution which is not an optimal scenario for SVT.

We will make sure to answer to this issue (with / without a PR) before updating the presets and it will not be combined with other PRs and I believe we have been quite disciplined on this based on the feedback we got.

**Comment by JacksonDagger** [comment]
I traced this back to commit a4aea0a in #367. This issue is resolved by #426.
